--- gforth-0.6.2/configure.in.orig	2003-12-18 23:35:27.000000000 +0100
+++ gforth-0.6.2/configure.in	2003-12-18 23:37:34.098199976 +0100
@@ -195,7 +195,7 @@
 	power*|rs6000)
 		machine=power
 		$srcdir/mkinstalldirs arch/power
-		AC_CHECK_FUNC(_sync_cache_range,true,AC_LIBOBJ(../arch/power/_sync_cache_range))
+		AC_CHECK_FUNC(_sync_cache_range,true,[AC_LIBOBJ(../arch/power/_sync_cache_range)])
 		#long long is broken on (at least) gcc-2.95.* for PPC
 		test x$ac_cv_sizeof_long_long = x && ac_cv_sizeof_long_long=0
 		;;
@@ -472,7 +472,7 @@
 AC_CHECK_FUNCS(ftello dlopen sys_siglist getrusage)
 AC_CHECK_TYPES(stack_t,,,[#include <signal.h>])
 AC_DECL_SYS_SIGLIST
-AC_CHECK_FUNC(getopt_long,true,AC_LIBOBJ(getopt); AC_LIBOBJ(getopt1))
+AC_CHECK_FUNC(getopt_long,true,[AC_LIBOBJ(getopt) AC_LIBOBJ(getopt1)])
 AC_CHECK_FUNCS(expm1 log1p)
 AC_REPLACE_FUNCS(rint ecvt)
 dnl No check for select, because our replacement is no good under
