--- gforth-0.7.3/configure.in.orig	2014-06-14 00:52:10.408636383 +0200
+++ gforth-0.7.3/configure.in	2014-07-14 20:19:01.629336298 +0200
@@ -250,9 +250,6 @@
 	i*86)
 		machine=386
 		CFLAGS="$CFLAGS -fomit-frame-pointer -fforce-addr"
-		CFLAGS_1="$CFLAGS"
-		CFLAGS="$CFLAGS -march=pentium"
-		AC_TRY_COMPILE(,,,CFLAGS="$CFLAGS_1 -m486")
 		;;
 	x86_64)
 		case $CC
@@ -260,8 +257,6 @@
 		    *-m32*)
 		    	machine=386
 			CFLAGS="$CFLAGS -fomit-frame-pointer -fforce-addr"
-			CFLAGS_1="$CFLAGS"
-			CFLAGS="$CFLAGS -march=athlon64"
 			;;
 		    *)
 		    	machine=amd64
