--- gforth-0.7.0/configure.in.orig	2008-11-02 18:14:57.000000000 +0100
+++ gforth-0.7.0/configure.in	2008-11-04 23:15:32.189552084 +0100
@@ -236,14 +236,11 @@
 		;;
 	i486)
 		machine=386
-		CFLAGS="$CFLAGS -fomit-frame-pointer -fforce-addr -m486"
+		CFLAGS="$CFLAGS -fomit-frame-pointer -fforce-addr"
 		;;
 	i*86)
 		machine=386
 		CFLAGS="$CFLAGS -fomit-frame-pointer -fforce-addr"
-		CFLAGS_1="$CFLAGS"
-		CFLAGS="$CFLAGS -march=pentium"
-		AC_TRY_COMPILE(,,,CFLAGS="$CFLAGS_1 -m486")
 		;;
 	x86_64)
 		case $CC
@@ -251,8 +248,6 @@
 		    *-m32*)
 		    	machine=386
 			CFLAGS="$CFLAGS -fomit-frame-pointer -fforce-addr"
-			CFLAGS_1="$CFLAGS"
-			CFLAGS="$CFLAGS -march=athlon64"
 			;;
 		    *)
 		    	machine=amd64
