--- gforth-0.6.1/doc/gforth.ds.orig	2003-03-11 22:15:16.000000000 +0100
+++ gforth-0.6.1/doc/gforth.ds	2003-05-11 01:47:22.000000000 +0200
@@ -77,9 +77,9 @@
 @end quotation
 @end copying
 
-@dircategory Software development
+@dircategory Programming languages:
 @direntry
-* Gforth: (gforth).             A fast interpreter for the Forth language.
+* Gforth: (gforth).			A fast Forth language interpreter
 @end direntry
 @c The Texinfo manual also recommends doing this, but for Gforth it may
 @c  not make much sense
@@ -9193,8 +9193,8 @@
 lower addresses. A few primitives allow an efficient implementation:
 
 
-doc-@local#
-doc-f@local#
+doc-@@local#
+doc-f@@local#
 doc-laddr#
 doc-lp+!#
 doc-lp!
@@ -9210,8 +9210,8 @@
 appropriate:
 
 
-@c doc-compile-@local
-@c doc-compile-f@local
+@c doc-compile-@@local
+@c doc-compile-f@@local
 doc-compile-lp+!
 
 
--- gforth-0.6.1/doc/vmgen.texi.orig	2003-03-09 16:06:43.000000000 +0100
+++ gforth-0.6.1/doc/vmgen.texi	2003-05-11 02:07:00.000000000 +0200
@@ -27,9 +27,9 @@
 @end quotation
 @end copying
 
-@dircategory Software development
+@dircategory Programming tools:
 @direntry
-* Vmgen: (vmgen).               Virtual machine interpreter generator
+* Vmgen: (vmgen).			Virtual machine interpreter generator
 @end direntry
 
 @titlepage
